hbase_sql/src/main/java/com/splicemachine/hbase/BackupUtils.java

```diff
@@ -14,16 +14,26 @@
 
 package com.splicemachine.hbase;
 
+import com.splicemachine.EngineDriver;
+import com.splicemachine.SpliceKryoRegistry;
 import com.splicemachine.access.HConfiguration;
+import com.splicemachine.access.api.PartitionAdmin;
+import com.splicemachine.access.api.PartitionFactory;
 import com.splicemachine.access.configuration.HBaseConfiguration;
+import com.splicemachine.access.hbase.HBaseConnectionFactory;
 import com.splicemachine.backup.BackupMessage;
 import com.splicemachine.backup.BackupMessage.BackupJobStatus;
 import com.splicemachine.backup.BackupRestoreConstants;
 import com.splicemachine.db.iapi.error.StandardException;
 import com.splicemachine.db.iapi.sql.dictionary.BackupDescriptor;
+import com.splicemachine.encoding.MultiFieldEncoder;
 import com.splicemachine.pipeline.Exceptions;
+import com.splicemachine.si.api.data.TxnOperationFactory;
 import com.splicemachine.si.constants.SIConstants;
 import com.splicemachine.si.impl.driver.SIDriver;
+import com.splicemachine.storage.DataPut;
+import com.splicemachine.storage.EntryEncoder;
+import com.splicemachine.storage.Partition;
```

Comment:
Cleanup imports, it doesn't look like they are needed.

Reply:
Fixed in commit https://github.com/splicemachine/spliceengine/pull/5103/commits/81376b4b31305ebc1dd7946beaccb4cc299844a1